edit/networking.k8s.io.ingress/index.vue

```diff
@@ -162,17 +162,10 @@ export default {
     },
 
     willSave() {
-      const defaultRule = this.value.spec.rules.filter(rule => rule.asDefault)[0];
-      const defaultBackend = defaultRule?.http?.paths[0]?.backend;
-      const nonDefaultRules = this.value.spec.rules.filter(rule => !rule.asDefault);
-
-      nonDefaultRules.forEach(rule => delete rule.asDefault);
-      this.value.spec.rules = nonDefaultRules;
-
-      if (defaultBackend ) {
-        this.$set(this.value.spec, 'backend', defaultBackend);
+      if (this.value?.spec?.backend && (!this.value?.spec?.backend?.serviceName || !this.value?.spec?.backend?.serviceName)) {
```

Comment:
same condition twice?

Reply:
second was supposed to be servicePort. Fixed.